app/controllers/alchemy/admin/essence_videos_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Alchemy
+  module Admin
+    class EssenceVideosController < Alchemy::Admin::BaseController
```

Comment:
now that we had a video essence we can remove the custom one that @dancristianb added 🤔

Reply:
Yep, I vote we track this under a cooldown card so we don't overload this PR 👍